applications/nrf5340_audio/src/audio/CMakeLists.txt

```diff
@@ -6,7 +6,11 @@
 
 target_sources(app PRIVATE
 	       ${CMAKE_CURRENT_SOURCE_DIR}/audio_system.c
-	       ${CMAKE_CURRENT_SOURCE_DIR}/streamctrl.c
 	       ${CMAKE_CURRENT_SOURCE_DIR}/audio_datapath.c
 	       ${CMAKE_CURRENT_SOURCE_DIR}/sw_codec_select.c
 )
+
+if (CONFIG_BT_BAP_BROADCAST_SOURCE)
```

Comment:
Discuss. We can use the GATEWAY and CONFIG_TRANSPORT_BIS defines. What about BAP vs CAP defines?

Reply:
I think using the CONFIG_BT_BAP defines are a good solution now.
And then transition away from using the CONFIG_AUDIO_DEV and CONFIG_TRANSPORT configs.